packages/xo-web/src/xo-app/backup-ng/index.js

```diff
@@ -141,6 +141,36 @@ class JobsTable extends React.Component {
         component: SchedulePreviewBody,
         name: _('jobSchedules'),
       },
+      {
+        itemRenderer: job => {
+          const { concurrency, offlineSnapshot } = job.settings[''] || {}
+
+          return (
+            <div>
+              {concurrency > 0 && (
+                <span>
+                  <label>
```

Comment:
Why a `label`?

Reply:
Just for structure label -> value, i will remove it.

Reply:
A `label` is a dedicated for inputs in forms.